docs/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md

```diff
@@ -99,6 +100,9 @@ Defines one or more switches used by the <xref:System.AppContext> class to provi
 |`Switch.System.ServiceModel.`<br/>`DisableUsingServicePointManagerSecurityProtocols`|Along with `Switch.System.Net.DontEnableSchUseStrongCrypto`, determines whether WCF message security uses TLS 1.1 and TLS 1.2.|.NET Framework 4.7 |    
 |`Switch.System.ServiceModel.`<br/>`UseSha1InMsmqEncryptionAlgorithm`|Determines whether the default message signing algorithm for MSMQ messages in WCF is SHA1 or SHA256.|.NET Framework 4.7.1|
 |`Switch.System.ServiceModel.`<br/>`UseSha1InPipeConnectionGetHashAlgorithm`|Controls whether WCF uses a SHA1 or a SHA256 hash to generate random names for named pipes.|.NET Framework 4.7.1|
+|`Switch.System.ServiceModel.`<br/>`DisableExplicitConnectionCloseHeader`|When using the HTTP transport with a self hosted service, settings this value to true causes WCF to ignore an application adding the `Connection: close` header to the response headers for a request. Setting this value to false enables adding the `Connection: close` header to the response headers which results in the closing of the request socket after a response has been sent.|.NET Framework 4.6|
```

Comment:
- This isn't in alphabetical order.
- self hosted --> self-hosted
- settings --> setting 
- true --> \`true\`
- false --> \`false\`
- headers which results in the closing of --> headers, which results in closing